components/process.rst

```diff
@@ -100,6 +100,38 @@ with a non-zero code)::
         echo $exception->getMessage();
     }
 
+.. tip::
+
+    .. versionadded:: 3.3
+        The ability to define commands as arrays of arguments was introduced in
+        Symfony 3.3.
+
+    Using array of arguments is the recommended way to define commands. This
+    saves you from any escaping and allows sending signals seamlessly
+    (e.g. to stop processes before completion.)::
+
+        $process = new Process(array('/path/command', '--flag', 'arg 1', 'etc.'));
+
+    If you need use stream redirections, conditional execution, or any other
```

Comment:
> need to use

Reply:
thanks, fixed in 2c5579be19da893059b51ff6d01ce4d1ec625c82